src/main/kotlin/org/kamiblue/client/module/modules/render/Xray.kt

```diff
@@ -0,0 +1,33 @@
+package org.kamiblue.client.module.modules.render
+
+import net.minecraft.block.state.IBlockState
+import org.kamiblue.client.module.Category
+import org.kamiblue.client.module.Module
+import org.kamiblue.client.setting.settings.impl.collection.CollectionSetting
+
+internal object Xray : Module(
+    name = "Xray",
+    description = "Lets you see through blocks",
+    category = Category.RENDER
+) {
+    private val defaultVisibleList = linkedSetOf("minecraft:diamond_ore", "minecraft:iron_ore", "minecraft:gold_ore")
+
+    val showFluids by setting("Show Fluids", false)
+
+    val visibleList = setting(CollectionSetting("Visible List", defaultVisibleList, { false }))
```

Comment:
Could you make this setting private and instead add methods that validate if it's a valid block or not when adding? Also I'm not sure if 0 visibility is a good idea or if it even does anything

Reply:
Nevermind I just realized that the command uses block args, disregard this comment